### PR TITLE
Tobin gh601 clear attach count

### PR DESCRIPTION
**Pull request by tobine**
Addresses missing check as requested in #601

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -8926,13 +8927,19 @@ CmdBeginRenderPass(VkCommandBuffer commandBuffer, const VkRenderPassBeginInfo *p
...
+                                                              "must therefore be greater than or equal to %u.",
+                    pRenderPassBegin->clearValueCount, reinterpret_cast<uint64_t &>(renderPass), clear_op_count, clear_op_count);
+            }
             skipCall |= VerifyRenderAreaBounds(dev_data, pRenderPassBegin);
             skipCall |= VerifyFramebufferAndRenderPassLayouts(dev_data, pCB, pRenderPassBegin);
             skipCall |= insideRenderPass(dev_data, pCB, "vkCmdBeginRenderPass");
-            if (renderPass) {
-                skipCall |= ValidateDependencies(dev_data, framebuffer, renderPass);
-            }
+            skipCall |= ValidateDependencies(dev_data, framebuffer, renderPass);
```
Comment (lentinem) [inline layers/core_validation.cpp]: If there is no render pass why are we validating dependencies.

Comment (tobine) [inline layers/core_validation.cpp]: See line 8899 above, we already checked to make sure we had renderPass so this check was redundant.

**Comment by lentinem** [comment]
+1

**Comment by tobine** [comment]
cherry-picked and pushed to master

**Pull request merged**